KarhooUISDK/Screens/PassengerDetails/PassengerDetailsViewController.swift

```diff
@@ -138,6 +139,7 @@ final class PassengerDetailsViewController: UIViewController, PassengerDetailsVi
                                             isOptional: false,
                                             accessibilityIdentifier: KHPassengerDetailsViewID.lastNameInputView)
         inputView.delegate = self
+        inputView.setBackgroundColor(.white)
```

Comment:
I wasn't sure if it will be correct color for dark mode so I changed it to just white and wanted to decided about dark mode appearance in proper ticket. In this way ( with hardcoded .white) we for sure take a closer look at this in future. But if you want, I can change it to background2 and maybe, it will be ok in dark mode too.

Reply:
ok, I changed it to background2. Just keep in mind that currently on develop background2 is not white. So it will be on once we merged one step payment branch to develop

Reply:
We'll release only after we merge 1-step payment, so we should be good in theory. We should see any issue in light mode when we do the pre-release regression testing, and we'll cover dark mode when he handle the whole screen UI.